.github/ISSUE_TEMPLATE/page-request.yml

```
@@ -0,0 +1,35 @@
+name: 📄 Page request
+title: "page request: "
+description: Page request template
+labels: new command
+body:
+  - type: textarea
+    attributes:
+      label: Command description
+      description: Describe a command you want to be summarized
```

```suggestion
      description: Describe a command you want to be summarized.
```

Please apply this change throughout.